file-browser/package.json

```diff
@@ -22,17 +22,19 @@
   },
   "private": true,
   "dependencies": {
+    "electron-is-dev": "^0.2.0",
+    "eywa": "git+https://github.com/escherpad/eywa.git",
     "layout-components": "^2.7.1",
     "luna": "^1.5.0",
-    "luna-saga": "^3.1.0",
+    "luna-saga": "^3.0.1",
+    "npm": "^5.1.0",
```

Comment:
Not necessary. NPM module is included by accident, when we are fixing the npm itself.